catalogd/internal/storage/localdir_test.go

```diff
@@ -238,7 +238,7 @@ func TestLocalDirServerHandler(t *testing.T) {
 		{
 			name:               "Server returns 404 when non-existent catalog is queried",
 			expectedStatusCode: http.StatusNotFound,
-			expectedContent:    "404 Not Found",
+			expectedContent:    "404 page not found",
```

Comment:
Oh this is interesting. I actually don't love this (because we aren't serving pages), but I also think it doesn't actually matter at all. It's just my OCD.

Reply:
Right. I guess it's something we have to live with if we want to `serveFile`

Reply:
What is the motivation for using serveFile instead of ServeContent as it is? Is there any specific reason to prefer one over the other?

Reply:
Might be worth looking at the implementations to see if there really is a significant difference. If it's just a difference of who opens the file, then maybe sticking with `ServeContent` is better because then 404 errors are consistent.

Reply:
I took a look. `ServeFile` has a bunch of handling for directories, and then eventually calls `serveContent`. I think we close this and keep things as they are?

Sorry for the noise @anik120 I know I was the one that originally suggested this change.